src/main/scala/com/github/sbt/sbom/BomSbtPlugin.scala

```diff
@@ -46,6 +58,10 @@ object BomSbtPlugin extends AutoPlugin {
       bomFileName := bomFileNameSetting.value,
       bomSchemaVersion := defaultSupportedVersion.getVersionString,
       includeBomSerialNumber := false,
+      includeBomTimestamp := false,
+      includeBomToolVersion := true,
+      includeBomHashes := true,
+      enableBomSha3Hashes := true,
```

Comment:
I wonder if we should default this to `false`, to be more reproducible by default...

Reply:
I also thought about this but java 8 is mostly end of life anyway and the sha3 family will be pretty important in the future and should probably be enabled by default sooner than later. The gradle cyclonedx plugin also does not filter out hashes.